src/test/java/build/buildfarm/worker/util/WorkerTestUtils.java

```diff
@@ -218,7 +218,7 @@ public static void assertFilesExistExactly(Path root, List<Path> expectedFiles)
     List<Path> listedPaths = listFilesRec(root);
     for (Path filePath : listedPaths) {
       assertWithMessage("Path not match prefix of any expected file: " + filePath)
-          .that(expectedFiles.stream().anyMatch(p -> p.startsWith(p)))
+          .that(expectedFiles.stream().anyMatch(p -> p.startsWith(filePath)))
```

Comment:
Just seems like a bug and unrelated to method lambda specification

Reply:
Moved to https://github.com/bazelbuild/bazel-buildfarm/pull/1785